### ISSUE TITLE
[Bug] Language does not change in the legal screen

### ISSUE THREAD

**Issue by Chaos-Shadow** [ThirteenAG/GTAIV.EFLC.FusionFix#475]
### Game version this happens in

v1.2.0.59

### What area of the game

When I start the game, the legal screen is in german but when I´m in the main menu the language is in english.
Here is another person having the same issue https://steamcommunity.com/app/12210/discussions/0/2243300286287657935/
This bug is not very well known when you have your windows display language set to english but all other languages will be displayed by default in GTAIV and changing it won´t fix the intro text because the game uses your windows language on top of the language ingame.

### What should happen

The legal screen should be fixed so it displays the correct language like in this mod 
https://github.com/gennariarmando/iv-language-loader
Basically it overrides the windows default language and uses the one from the mod.
Also I know I could use this mod but I want this mod to be part of fusion fix as it is an essential fix

### Checklist

- [X] Test in the [latest git build](https://ci.appveyor.com/api/projects/ThirteenAG/gtaiv-eflc-fusionfix/artifacts/GTAIV.EFLC.FusionFix.zip) in case it's already fixed.
- [X] [Search for other reports](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/search?q=my+issue&type=issues) of the same issue.
- [X] Include logs or screenshots of issue.

**Comment by Tomasak** [comment]
language is controlled by option set per game in either RGL or Steam game properties

**Comment by Chaos-Shadow** [comment]
> language is controlled by option set per game in either RGL or Steam game properties

I know, but It does not work for me.

**Comment by Tomasak** [comment]
i cannot reproduce this. I tried english windows account and setting game to italian and it was correctly in italia. Well either way, either report this to Rockstar support or use the language loader
